snap/local/bin/journald-ingester

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+ENVFILE="${SNAP_DATA}/journald-env"
+CONFFILE="${SNAP_DATA}/journald-config"
+
+if [ ! -f "${CONFFILE}" ] ; then
+    echo "Journald configuration not available"
+    exit 0
+fi
+
+if [ ! -f "${ENVFILE}" ] ; then
+    echo "Journald environment not available"
+    exit 0
+fi
+
+. ${ENVFILE}
+
+if [ -z ${AXIOM_DATASET} ]; then
+    echo "Configuration not complete: ${ENVFILE}"
+    exit 0
+fi
+
+exec journalctl --output=json --no-page --follow | ${SNAP}/bin/axiom ingest -C ${CONFFILE} ${AXIOM_DATASET}
```

Comment:
Think that should probably configure the timestamp. Not sure which to use though. Here is sample JSON:

```json
{
  "MESSAGE": " watchly : TTY=pts/3 ; PWD=/home/watchly/cli ; USER=root ; COMMAND=/usr/bin/snap logs axiom",
  "PRIORITY": "5",
  "SYSLOG_FACILITY": "10",
  "SYSLOG_IDENTIFIER": "sudo",
  "SYSLOG_TIMESTAMP": "Sep 17 18:57:55 ",
  "_AUDIT_LOGINUID": "1001",
  "_AUDIT_SESSION": "29",
  "_BOOT_ID": "9f896132e65d4f6a9e39c74fcb517624",
  "_CAP_EFFECTIVE": "ffffffffff",
  "_CMDLINE": "sudo snap logs axiom",
  "_COMM": "sudo",
  "_EXE": "/usr/bin/sudo",
  "_GID": "1001",
  "_HOSTNAME": "watchly-ubuntu",
  "_MACHINE_ID": "973666a9aa584292904535665a5dcc7a",
  "_PID": "3674245",
  "_SELINUX_CONTEXT": "unconfined\n",
  "_SOURCE_REALTIME_TIMESTAMP": "1631905075342103",
  "_SYSTEMD_CGROUP": "/user.slice/user-1001.slice/session-29.scope",
  "_SYSTEMD_INVOCATION_ID": "42cf102b9dbc470bbe508e7bd629465a",
  "_SYSTEMD_OWNER_UID": "1001",
  "_SYSTEMD_SESSION": "29",
  "_SYSTEMD_SLICE": "user-1001.slice",
  "_SYSTEMD_UNIT": "session-29.scope",
  "_SYSTEMD_USER_SLICE": "-.slice",
  "_TRANSPORT": "syslog",
  "_UID": "0",
  "__CURSOR": "s=082f2cfa5dce4810ae7233b2a747c7e8;i=68e7;b=9f896132e65d4f6a9e39c74fcb517624;m=2e71ff03354;t=5cc358252ab97;x=4e0cfae04d43a759",
  "__MONOTONIC_TIMESTAMP": "3191696536404",
  "__REALTIME_TIMESTAMP": "1631905075342231"
}
```

Reply:
I think we should explicitly specify the timestamp using `--timestamp-field=SYSLOG_TIMESTAMP`. `SYSLOG_TIMESTAMP` also seems like the best candidate. We should make sure to pass the correct `--timestamp-format` flag as well. From `axiom help ingest`:

```
The timestamp format can be configured by specifying a pattern with the reference date:

	Mon Jan 2 15:04:05 -0700 MST 2006

Omitted elements in the pattern are treated as zero or one as applicable. See the Go reference documentation for examples: https://pkg.go.dev/time#pkg-constants
```

Hint: Syslog by RFC5424 specifies the `TIMESTAMP` as RFC3339 with some more restrictions: https://datatracker.ietf.org/doc/html/rfc5424#section-6.2.3. However, the one in the sample JSON message looks nothing like RFC3339.